src/webview/create-component/pages/fromLocalCodebase.tsx

```diff
@@ -157,81 +163,10 @@ function RecommendedDevfile(props: RecommendedDevfileProps) {
                 <Button variant='text' onClick={() => { props.setCurrentView('devfileSearch') }}>
```

Comment:
We can look into hooking it up later, but if you modify the `props` that you pass to the `<DevfileSearch />` component, you can skip the modal where you pick the starter project.